gcc/testsuite/rust.test/fail_compilation/array_empty_list.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    let arr = [];
```

Comment:
With the new changes to the testsuite @dkm has done you should be able to add a dg-error line here if you rebase this PR.

Reply:
Cool. I'll do that.